Source/Core/Core/PowerPC/JitArm64/JitArm64_Branch.cpp

```diff
@@ -208,21 +217,23 @@ void JitArm64::bcctrx(UGeckoInstruction inst)
   gpr.Flush(FlushMode::All, ARM64Reg::INVALID_REG);
   fpr.Flush(FlushMode::All, ARM64Reg::INVALID_REG);
 
+  ARM64Reg WB = ARM64Reg::INVALID_REG;
   if (inst.LK_3)
   {
-    ARM64Reg WB = gpr.GetReg();
+    WB = gpr.GetReg();
     MOVI2R(WB, js.compilerPC + 4);
     STR(IndexType::Unsigned, WB, PPC_REG, PPCSTATE_OFF_SPR(SPR_LR));
-    gpr.Unlock(WB);
   }
 
   ARM64Reg WA = gpr.GetReg();
 
   LDR(IndexType::Unsigned, WA, PPC_REG, PPCSTATE_OFF_SPR(SPR_CTR));
   AND(WA, WA, LogicalImm(~0x3, 32));
 
-  WriteExit(WA, inst.LK_3, js.compilerPC + 4);
+  WriteExit(WA, inst.LK_3, js.compilerPC + 4, WB);
```

Comment:
At first glance it looks like there's a bug here because you're using `WB` regardless of whether `inst.LK_3` is true... But upon closer inspection, `WriteExit` ignores the parameter if `inst.LK_3` isn't true. I don't really like it, but I'm not sure if adding an if statement or something really would be neater, so I'll leave it to your judgment.